### DISCUSSION TITLE
Compare 2 folders and retrieve exact folders and filenames when re-organization performed on source

### DISCUSSION THREAD

**Discussion by inthegardendog** [WinMerge/winmerge#2366] [Ideas]
We often have some comparison use cases where source files have been renamed or moved in the tree structure, but content is the same.
When doing comparison using file content, it would be quite useful to identify, given 2 folders, which files have been moved and renamed, and propose updates to re-synchronize the 2 folders when only file path location or filename have been modified, based on non-modification of content as input.

**Comment by otzen** [comment]
Need similar feature too.
I would suggest to do something like git, that can detect not only moved/cloned files, but also files that was moved, and edited.
The moved and edited case is based on files still being like 90% equal.

If WinMerge shall reimplement this algorithm or use Git to do the analysis is of no difference to me.
